### ISSUE TITLE
[Feature Request] Pop out window enhancement - Always on top & transparency

### ISSUE THREAD

**Issue by nategasm** [speech-translator-ext/speech-translator-readme#38]
Hi, I'm currently using Firefox as my main browser and I understand it's currently difficult to port this extension to it. 

As a work around I run Edge in the background with the extension's pop out window. However, you need to be mindful where to place the window since it is a separate browser. It would be really useful if there was an option to set the pop out window to always be on top, that way it can be placed anywhere we want on top of Firefox or other windows without needing to worry about losing window focus. In addition to this it would be good if transparency could also be applied to the pop out window.

PS. Thanks for your work on this

**Comment by kappaflow** [comment]
>  set the pop out window to always be on top
> if transparency could also be applied to the pop out window

Unfortunately, both these things are technically impossible to do. However, I use Google Chrome for browsing and Edge for speech translation. I run Speech Translator in a pop-up window and place it on the second screen.

Another workaround: you can activate Streaming Mode, then open that link in Firefox or basically anywhere (even on your mobile phone). When combined with OBS, I think you can even display it in a video player or any app that can display video feed from a video capture device (like a webcam).

**Comment by kappaflow** [comment]
Yes, you can use 3rd party tools for that, but it will be still challenging to have transparency for the background only. Not sure if its even possible.
